src/core_plugins/tile_map/public/base_maps_visualization.js

```diff
@@ -88,6 +87,8 @@ export function BaseMapsVisualizationProvider(serviceSettings) {
       options.center = centerFromUIState ? centerFromUIState : this.vis.params.mapCenter;
 
       this._kibanaMap = new KibanaMap(this._container, options);
+      this._kibanaMap.setMinZoom(0);//use a default
```

Comment:
why are this defaults hardcoded here and not read from vis.params ?

Reply:
These settings are driven by the base-layers and those are read from the EMS-manifests. These are just placeholders when we create the map, but they get overridden as baselayers are configured.

I did create variables for them.